src/main/java/com/amazon/sqs/javamessaging/SQSConnection.java

```diff
@@ -179,26 +179,26 @@ public QueueSession createQueueSession(boolean transacted, int acknowledgeMode)
      *             transaction and acknowledge mode.
      */
     @Override
-    public Session createSession(boolean transacted, int acknowledgeMode) throws JMSException {
+    public Session createSession(boolean transacted, int sessionMode) throws JMSException {
```

Comment:
Any particular reason to change the variable naming here?

Reply:
ohhh, right, something went sideways from my side, just checked the signature of the method, I will revert it back tomorrow.

Reply:
@armogur thank you! And also be aware that, you should point your fork to merge awslabs:v2 branch as opposed to the master branch. The master branch is intended for java v1 and v2 is for java v2. I will work on upgrading for java v1 while you already took care of the v2. I left some minor comments there, the majorities are format related. Please help us retain a good and uniformed package code style and convention. thanks a lot.

Reply:
fixed, please check.